File: 1143.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var a = Number(lines.shift());
for(var i = 1; i < a+1;i++){
    var b = Math.pow(i, 2)
    var c = b * i
    console.log(`${i} ${b} ${c}`)};

File: 1143.java
import java.util.Scanner;
public class Main {
public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int a = input.nextInt();
        for(int i = 1; i < a+1; i++) {
                int b = (int) Math.pow(i,2);
                int c = b * i;
                System.out.println(i + " " + b + " " + c);
        }
}
}
